Lookup Results.hidden-tmLanguage

```diff
@@ -9,7 +9,7 @@
 	<array>
 		<dict>
 			<key>match</key>
-			<string>(.*):$</string>
+			<string>^(((?!scope: ).)*):$</string>
```

Comment:
Can you explain why you changed this please? I'm not aware of a problem with this right now, so I'm not sure what this is trying to fix.

Reply:
If line with symbol ends with ':' it is catched by this regex (seen below, result treated like filename), fix prevents it.
![zrzut ekranu 2018-04-08 17 15 21](https://user-images.githubusercontent.com/7755608/38471909-15a98f0a-3b78-11e8-8929-579b1ccde003.png)